setup.md

```diff
@@ -21,6 +21,25 @@ To install and run Flutter, your development environment must meet these minimum
 * **Tools**: Flutter depends on these command-line tools being available on your environment.
   * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
 
+## Get the Flutter SDK
+
+To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
+
+<pre>
+$ git clone https://github.com/flutter/flutter.git -b alpha
+$ export PATH=`pwd`/flutter/bin:$PATH
+</pre>
+
+To complete the setup and to validate that you’ve set everything up correctly, run the following
+command:
```

Comment:
This sentence is out of place now.  We haven't installed Xcode yet, so we won't be able to verify that we've set up everything correctly.  Maybe say that this command will show you what needs to be installed?